src/pages/Admin/Venue/DetailsValidationSchema.ts

```diff
@@ -46,53 +48,34 @@ const createFileSchema = (name: string, required: boolean) =>
       }
     );
 
-const urlIfNoFileValidation = (fieldName: string) =>
-  Yup.string().when(
-    fieldName,
-    (file: FileList | undefined, schema: Yup.MixedSchema<FileList>) =>
-      file && file.length > 0
-        ? schema.notRequired()
-        : schema.required("Required")
-  );
```

Comment:
[note] This is exactly the same as the version in `src/pages/Admin/Details/ValidationSchema.ts` that is now imported here.